### PR TITLE
wxGUI ImportDialog: Improve output map name existence validation

### PR DESCRIPTION
**Pull request by tmszi**
Improve output map name existence validation for `GdalImportDialog`, `OgrImportDialog`, `DxfImportDialog` class.

Import raster data dialog with the improved validation:

import_dialog_output_name_validation

### CODE REVIEW

File: gui/wxpython/modules/import_export.py [hunk 1]
```diff
@@ -241,6 +243,36 @@ def _getBlackListedFlags(self):
...
+    def _nameValidationFailed(self, layers_list):
+        """Output map name validation callback
+
+        :param layers_list: LayersList class instance
+        """
+        if isinstance(layers_list.output_map, list):
+            maps = [
+                '<{}>'.format(m) for m in layers_list.output_map]
+            message = _(
+                "Output maps names %(names)s exist. "
```
Comment (petrasovaa) [inline gui/wxpython/modules/import_export.py:255]: It should be "Output map names" I think.

File: gui/wxpython/gui_core/widgets.py [hunk 2]
```diff
@@ -979,6 +980,55 @@ def Select(self):
...
+        mapset = grass.gisenv()['MAPSET']
+        maps = grass.list_grouped(type=self._condition)[mapset]
+
+        # Check all selected layers
+        if validate_all:
+            outputs = []
+            data = win.GetLayers()
+
+            if data is None:
+                return
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/widgets.py:1012]: Probably it should be `return False` here?

File: gui/wxpython/modules/import_export.py (hunk 1 above)
Comment (tmszi) [reply gui/wxpython/modules/import_export.py]: I fixed it.

File: gui/wxpython/gui_core/widgets.py (hunk 2 above)
Comment (tmszi) [reply gui/wxpython/gui_core/widgets.py]: I added it.

**Pull request merged**